src/tests/ftest/util/ec_utils.py

```diff
@@ -133,3 +142,167 @@ def ior_read_dataset(self, parity=1):
                 self.start_ior_load(operation='Read', percent=1,
                                     create_cont=False)
                 con_count += 1
+
+class ErasureCodeSingle(TestWithServers):
+    # pylint: disable=too-many-ancestors
+    """
+    Class to used for EC testing for single type data.
+    """
+
+    def __init__(self, *args, **kwargs):
+        """Initialize a TestWithServers object."""
+        super().__init__(*args, **kwargs)
+        self.server_count = None
+        self.set_online_rebuild = False
+        self.rank_to_kill = None
+        self.container = []
+
+    def setUp(self):
+        """Set up each test case."""
+        # Start the servers and agents
+        super().setUp()
+        self.server_count = len(self.hostlist_servers) * 2
```

Comment:
`2` should be replaced by `engine_count = self.server_managers[0].get_config_value("engines_per_host")`

Reply:
I will change